lib/twilio.rb

```diff
@@ -0,0 +1,23 @@
+require 'twilio-ruby'
+
+class Twilio_sms
+
+  def initialize(dishes, total)
+    @dishes = dishes
+    @total = total
+  end
+
+  def send_text_message
+
+    account_sid= ENV['ACCOUNT_SID']
+    auth_token = ENV['AUTH_TOKEN']
+
+    @client = Twilio::REST::Client.new account_sid, auth_token
+
+    @client.account.sms.messages.create(
+      :from => ENV['TWILIO_NUM'],
+      :to => ENV['MY_NUM'],
+      :body => "Thank you! Your order is #{@dishes} with a total of #{@total}! Your order was placed and will be delivered before 18:52."
```

Comment:
Metrics/LineLength: Line is too long. [137/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)